test/functional/apps/settings/_index_pattern_results_sort.js

```diff
@@ -93,9 +93,11 @@ define(function (require) {
         });
 
         bdd.it('makelogs data should have expected number of fields', function () {
-          return settingsPage.getFieldsTabCount()
-          .then(function (tabCount) {
-            expect(tabCount).to.be('' + expectedFieldCount);
+          return common.tryForTime(5000, function () {
```

Comment:
I'm not sure how to do this at the moment.  We need tryForTime to return a value.  See the second issue here;
https://github.com/elastic/kibana/issues/5362